packages/jest-circus/src/legacy-code-todo-rewrite/jestAdapter.ts

```diff
@@ -101,7 +102,10 @@ const jestAdapter = async (
     globalConfig,
     testPath,
   });
-  return _addSnapshotData(results, snapshotState);
+
+  _addSnapshotData(results, snapshotState);
+
+  return deepCyclicCopy(results);
```

Comment:
I'd say perhaps a comment, and definitely `{keepPrototype: false}` would be nice to explain this?

Reply:
add suggestion? 😀

Reply:
50/50 if I get Prettier's brace spacing right :joy: